src/api/init.ts

```diff
@@ -172,6 +173,17 @@ export async function startApiServer(datastore: DataStore, chainId: ChainID): Pr
     })()
   );
 
+  //Setup routes for token metadata
+  app.use(
+    '/v1',
```

Comment:
Any reason why this isn't defined under the `/extended/v1` route?

Reply:
I wasn't aware of the convention. let me change it.